packages/deployment-service/package.json

```diff
@@ -12,7 +12,7 @@
     "@nestjs/platform-express": "^9.4.3",
     "@nestjs/typeorm": "^8.1.4",
     "@octokit/app": "^14.0.2",
-    "@reapit/connect-session": "^6.1.1",
+    "@reapit/connect-session": "^6.1.3",
```

Comment:
am sure I deleted this the other day. Connect session shouldn't be used by the deployment-service anymore